tapsend/send.go

```diff
@@ -590,6 +590,14 @@ func PrepareOutputAssets(ctx context.Context, vPkt *tappsbt.VPacket) error {
 		splitLocators = append(splitLocators, &locator)
 	}
 
+	// If we're creating a split commitment, there must be a split root.
+	// If none of the outputs was designated as the split root, then
+	// something was done incorrectly.
+	if rootLocator == nil {
+		return fmt.Errorf("cannot create split commitment without " +
```

Comment:
If we make this into a concrete error like `ErrNoRootLocator`, then in the future we can match against it within unit tests.